framework/CHANGELOG.md

```diff
@@ -4,7 +4,7 @@ Yii Framework 2 Change Log
 2.0.27 under development
 ------------------------
 
-- no changes in this release.
+- Enh #17549: Added `yii\db\ExpressionInterface` support to `yii\db\conditions\SimpleConditionBuilder` (razvanphp)
```

Comment:
That could be considered a bug.

Reply:
True, but didn't know for sure if it worked like this before 2.0.14 changes...